lib/pages/caregiver/plan_details_page.dart

```diff
@@ -25,141 +26,118 @@ class CaregiverPlanDetailsPage extends StatefulWidget {
 
 class _CaregiverPlanDetailsPageState extends State<CaregiverPlanDetailsPage> {
 	static const String _pageKey = 'page.caregiverSection.planDetails';
-	Mongo.ObjectId planID;
+
 
   @override
   Widget build(BuildContext context) {
-		Mongo.ObjectId planID = ModalRoute.of(context).settings.arguments;
-
 		return Scaffold(
 			body: Column(
 				crossAxisAlignment: CrossAxisAlignment.start,
 				children: [
-					AppHeader.widget(
-						title: '$_pageKey.header.title',
-						appHeaderWidget: ItemCard(
-							title: planID != null ? planID.toString() : "Sprzątanie pokoju",
-							subtitle: "Co każdy poniedziałek, środę, czwartek i piątek",
-							chips:
-							<Widget>[
-								AttributeChip.withIcon(
-									content: AppLocales.of(context).translate('page.caregiverSection.plans.content.tasks', {'NUM_TASKS': 1}),
-									color: Colors.indigo,
-									icon: Icons.layers
-								)
-							],
-						),
-						helpPage: 'plan_info',
-						popupMenuWidget: PopupMenuList(
-							lightTheme: true,
-							items: [
-								UIButton.ofType(ButtonType.edit, () => Navigator.of(context).pushNamed(AppPage.caregiverPlanForm.name, arguments: Mongo.ObjectId.parse('5f3ad3cf4ea005fba95da86b'))),
-								UIButton.ofType(ButtonType.delete, () => showBasicDialog(
-									context,
-									GeneralDialog.confirm(
-										title: AppLocales.of(context).translate('alert.deletePlan'),
-										content: AppLocales.of(context).translate('alert.confirmPlanDeletion'),
-										confirmText: 'actions.delete',
-										confirmAction: () => Navigator.of(context).pop(),
-										confirmColor: Colors.red
-									)
-								))
-							],
-						)
-					),
-					AppSegments(segments: _buildPanelSegments())
+					LoadableBlocBuilder<CaregiverTasksCubit>(
+						builder: (context, state) => AppSegments(segments: _buildPanelSegments(state)),
```

Comment:
Wrong, header needs to be above the AppSegments or we achieve this when scrolling:
![image](https://user-images.githubusercontent.com/8643919/92643922-d77c4e00-f2e2-11ea-8770-b00a733fee77.png)

Reply:
The same mistake is in ChildPlanInProgressPage, be sure to fix both!

Reply:
I already made task on Jira :)